src/main/fc/init.c

```diff
@@ -366,6 +365,8 @@ void init(void)
     pgResetAll();
 
 #if defined(STM32H7) && defined(USE_SDCARD_SDIO) // H7 only for now, likely should be applied to F4/F7 too
+    sdcardConfigMutable()->mode = SDCARD_MODE_SDIO;
```

Comment:
This should not be in here - if this config is needed it should be applied in the parameter group reset.

Reply:
Done